src/TurbulenceClosures/turbulence_closure_implementations/isopycnal_skew_symmetric_diffusivity.jl

```diff
@@ -172,7 +172,9 @@ end
     
     slope_x = - bx / bz
     slope_y = - by / bz
-    slope² = ifelse(bz < 0, zero(grid), slope_x^2 + slope_y^2)
+   
+    # in case of an 
```

Comment:
in case of what?

Reply:
of a stable buoyancy gradient (bz > 0), the slope is set to zero 
is this right though? should we taper the slope of the diffusivity?

Reply:
sorry the slope _or_ the diffusivity?

Reply:
not sure really
@glwagner ?